markdown/GameGuides/EldenRing.md

```diff
@@ -0,0 +1,98 @@
+
+Its a bit slower progress of course with the first method because sometimes, if not always, I tweak FoV while I am running the game at high DSR res because if your monitor has a 16:9 ratio, and you want to take the shot in 21:9, you will need to compose a bit on that DSR res and if you turned on Higher LODs from the tools along with ReShade it can be quite laggy. But there is also the other way of taking the high-res shots in this game.
+
+### Fullscreen Way
+@alert tip
+You can access the video settings even if your game is paused, while the free cam is on! This can be useful if you want to switch to different aspect ratios in fullscreen mode while not ruining your perfectly aligned shot. You just gotta do this with a keyboard and mouse if the camera control device is set to `Gamepad` in tools. If it's set to `KeyboardMouse`, then you can access the menu with the controller. If it's set to `Both` it will not work. Don't forget to turn the `HUD` back on otherwise you won't see the menu after pressing the `ESC`.
+@end
+
+Let's say that you are in 21:9, or 16:9, but you realised that 3:4 would suit you better for this moment, or 1:1, or you just want to experiment, so you press `ESC`, and navigate with a mouse (if the camera control device is set to `Gamepad` in tools) to video settings to pick the resolution of your choice and it will instantly switch to that DSR resolution! You can do this as many times as you want. Honestly, I never experienced crashes while doing this countless of times in one play session of 4 hours.
+
+It's really rare to happen, but keep in mind that this method sometimes won't work if you press the `Alt`+`Enter` or the `Windows` key for some reason or by accident. You will see the menu, but you won't be able to access it. So pay attention to that if you want to use this method straight away, or reset to refresh if the menu bug out, combine. But like I said it's rare, and it will only happen if you `Alt`+`Tab` too much while you are in a custom aspect ratio.
+
+## Resembling Hotsampling
+
+If you are dedicated, you can generate lower custom DSR resolutions with the tool only for composing, then do the [fullscreen](../GameGuides/EldenRing.htm#fullscreen-way) menu trick to go to much higher res in that aspect ratio to take the shot and also the same way to return to lower one. Avoid setting custom DSR resolutions too low, because it will crash your game. Generated resolutions shouldn't have lower width/height than your monitor's native one. In my case, I went with x1.5 for 21:9, and x2.25 for 3:4. Any lower caused my game to crash.
+
+Another helpful tip is to check other [tools](https://i.imgur.com/dRk02S8.png) that support hotsampling. This way, you can find aspect ratio resolutions offered for your monitor that can be used with the DSR tool.
+
+I recommend testing lots of custom DSR resolutions to find the best ones for your card.
+
+If you have RTX 4090, *don't think too much*.
```

Comment:
It's a guide, not a blog post, please leave these out of the guide 👍

Reply:
But.. ok :)